### ISSUE TITLE
“Background Items Added” notifications loop

### ISSUE THREAD

**Issue by philsherry** [Homebrew/homebrew-core#118814]
### `brew gist-logs <formula>` link OR `brew config` AND `brew doctor` output

```shell
✘ ✝  ~  🏠🔓  brew gist-logs memcached
Error: No logs.
 ✘ ✝  ~  🏠🔓  brew config
HOMEBREW_VERSION: 3.6.16
ORIGIN: https://github.com/Homebrew/brew
HEAD: 740f9ddcc620d4812770a0f12854ddd79c643874
Last commit: 2 days ago
Core tap ORIGIN: https://github.com/Homebrew/homebrew-core
Core tap HEAD: fa4e8e8dc5cfd189a6de1793164880d1c6ef968a
Core tap last commit: 2 hours ago
Core tap branch: master
HOMEBREW_PREFIX: /usr/local
HOMEBREW_CASK_OPTS: ["--fontdir=/Library/Fonts"]
HOMEBREW_CLEANUP_PERIODIC_FULL_DAYS: 3
HOMEBREW_CLEANUP_MAX_AGE_DAYS: 5
HOMEBREW_INSTALL_BADGE: 🫖
HOMEBREW_MAKE_JOBS: 6
Homebrew Ruby: 2.6.10 => /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/bin/ruby
CPU: hexa-core 64-bit kabylake
Clang: 14.0.0 build 1400
Git: 2.39.0 => /usr/local/bin/git
Curl: 7.85.0 => /usr/bin/curl
macOS: 13.1-x86_64
CLT: 14.2.0.0.1.1668646533
Xcode: 14.2
 ✝  ~  🏠🔓  brew doctor
Your system is ready to brew.
```

### Verification

- [X] My "`brew doctor` output" says `Your system is ready to brew.` and am still able to reproduce my issue.
- [X] I ran `brew update` and am still able to reproduce my issue.
- [X] I have resolved all warnings from `brew doctor` and that did not fix my problem.

### What were you trying to do (and why)?

I was trying to run `memcached` using Homebrew — it was already installed, but I’d recently upgraded macOS on this Mac.

### What happened (include all command output)?

The new-ish macOS “Background Items Added” notification adds multiple notifications for memcached, regardless of acceptance, not stopping until turned off in “Allow in the background” in Login Items. (Not the preferred outcome.)

![memcached-homebrew](https://user-images.githubusercontent.com/272769/208872224-859a59ac-5647-4a6e-a45d-b9812dc471ef.png)

### What did you expect to happen?

I expected it to quietly run in the background, like it had been all along.

### Step-by-step reproduction instructions (by running `brew` commands)

```shell
Install (via `brew install memcached`) and watch the notifications pile up, like the attached screenshot.

- OS/Distro: macOS Ventura
- Version of OS/distro: macOS 13.1 (22C65)
- Version of memcached: 1.6.17 (via `brew install memcached`)
- Hardware detail: 3 GHz 6-Core Intel Core i5
```

**Comment by SMillerDev** [comment]
I can't reproduce this by installing memcached.

**Comment by gromgit** [comment]
Seems like it's a known macOS issue, that was supposed to have been resolved in 13.1: https://piunikaweb.com/2022/12/09/macos-13-ventura-background-items-added-notification-issue/

@philsherry, see if you have a weird entry in your Login Items system setting, like this:
![image](https://user-images.githubusercontent.com/215702/209346334-ae17b133-f397-4f7e-8b26-715664488155.png)

If so, try the workaround mentioned in the above article.

**Comment by philsherry** [comment]
@gromgit …and there it is. I hadn’t heard of that macOS bug, so thanks for pointing that out. I will close this and glare silently in the direction of Cupertino. Thanks.
